pkgs/os-specific/darwin/apple-source-releases/Libsystem/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, appleDerivation, cpio, xnu, Libc, Libm, libdispatch, cctools, Libinfo
 , dyld, Csu, architecture, libclosure, CarbonHeaders, ncurses, CommonCrypto
 , copyfile, removefile, libresolv, Libnotify, libplatform, libpthread
-, mDNSResponder, launchd, libutil, hfs, darling, darwin-stubs }:
+, mDNSResponder, launchd, libutil, hfs, darling, darwin-stubs, ... }:
```

Comment:
```suggestion
, mDNSResponder, launchd, libutil, hfs, darling, darwin-stubs }:
```

Or what did you mean to add here?

Reply:
We explicitly overwrite all packages with `{ headersOnly = true; }` for Libsystem by default, it will input extra unexpected arguments. Using `...` can ignore them, and should not cause unnecessary rebuild if not use.  

https://github.com/NixOS/nixpkgs/blob/6cfc811f415638127d6d6952375a31c6cfe4459e/pkgs/os-specific/darwin/apple-source-releases/default.nix#L253-L259

Reply:
I don't think we should allow anything as an input. We rebuild everything here anyway.

Reply:
Sorry I don't quite get it, variadic attribute is widely used in nixpkgs, attributes which not stated here wouldn't even be evaluated, much less to cause rebuild.

Reply:
I have removed this, maybe bring it back with no `...` when updating libsystem next step.